### PR TITLE
220312-updating my homework [CHANGE REQUEST]

### PR DESCRIPTION
**Pull request by sepantra**
Dear teammates check my work plz :)

### CODE REVIEW

File: 1-homework/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# python-first Project
+# python-first Project
+Dear user :
+you may have python programme in your pc,if not you may download it from The official python website,
+after that you can push the F5 bottom and run the program.
```
Comment (Samcn21) [inline 1-homework/README.md]: The instruction is not very clear, you should mentioning step by step, what is official site for python?

where should I press F5?

more descriptions please!

File: 1-homework/index.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+"""
+Created on Thu Mar  3 17:27:08 2022
+
+@author: Gv Shop
+"""
+
+Names=["Sayid","Moeen","Setareh"]
+Roles=["Project Manager","Web Developer","Python Programmer"]
+
+new={ "Hello "+Names[i]:"Role:" +Roles[i] for i in range(3)}
```
Comment (Samcn21) [inline 1-homework/index.py]: I can't see "Role: Something" be inside brackets? it should show like this: e.g. "`Hello Setareh [Role: Python Programmer]`"

File: 1-homework/index.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import pprint
```
Comment (Samcn21) [inline 1-homework/index.py]: imports are usually in the beginning of the page not in the middle

File: README.md [hunk 4]
```diff
@@ -1,2 +0,0 @@
-# python-test
```
Comment (Samcn21) [inline README.md:1]: the instruction should be in this file, you should remove the one that you made in homework-1 folder

Comment (Samcn21) [reply README.md:1]: so when I go to your repo I can see the instructions right away:

image

**Review by Samcn21 — CHANGES_REQUESTED** [review]
Very good. 👍

File: 1-homework/index.py (hunk 3 above)
Comment (sepantra) [reply 1-homework/index.py]: you mean before every codes I should import something that I need?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,5 @@
...
-training myself
+# python-first Project
+
+-To rum thios project you should download python program from its official website (https://www.python.org/downloads/)
```
Comment (Samcn21) [inline README.md:3]: to run this project...

**Review by Samcn21 — APPROVED** [review]

**Pull request merged**